rumqttd/src/server/broker.rs

```diff
@@ -412,6 +421,25 @@ impl<P: Protocol + Clone + Send + 'static> Server<P> {
     }
 }
 
+/// Configures the Websocket connection to indicate the correct protocol
+/// by adding the "sec-websocket-protocol" with value of "mqtt" to the response header
+#[cfg(feature = "websocket")]
+struct WSCallback;
+#[cfg(feature = "websocket")]
+impl Callback for WSCallback {
+    fn on_request(
+        self,
+        _request: &Request,
+        mut response: Response,
+    ) -> Result<Response, ErrorResponse> {
+        response.headers_mut().insert(
+            "sec-websocket-protocol",
+            axum::http::HeaderValue::from_static("mqtt"),
```

Comment:
`async_tungstenite` also exports `http`, so can we use that here, just to be consistent? ( we can import it at top, or just use it here right away.
```rust
async_tungstenite::tungstenite::http::HeaderValue;
```